File: python/006.py
#!/usr/bin/env python

def main():
  """ this computes projecteuler.net problem 006 """

  sumsq = 0
  for i in range(1, 101):
    sumsq += i ** 2

  sqsum = (101 * 50) ** 2
  
  print sqsum -sumsq

  return 0

if __name__ == "__main__":
    main()

File: perl/006.pl
#!/usr/bin/perl -w
use strict;

my $sumsq = 0;
foreach my $i (1 .. 100) {
  $sumsq += $i*$i;
}

my $sqsum = (101 * 50) ** 2;
print $sqsum - $sumsq,"\n";
